airflow/operators/python.py

```diff
@@ -528,6 +537,9 @@ def __init__(
         templates_exts: list[str] | None = None,
         expect_airflow: bool = True,
         skip_on_exit_code: int | Container[int] | None = None,
+        index_urls: None | Collection[str] | str = None,
+        cache_venv: bool = False,
+        cache_path: str = "/tmp",
```

Comment:
Why not make this `cache_path: str | None = None` and disable cache on None?

Also a change of the argument name may be a good idea; it’s unclear from the current name what exactly is cached since pip and Python (on execution time) also has their own cache setup.

Reply:
As proposed I separated-out the caching option, a separate PR is made for this. As it builds-on top of small re-structuring (would not call it re-factoring :-D) I made a PR to _this_ feature branch. Target merge directory of the other PR would need to be switched to apache/airflow:main post merging this PR:

https://github.com/boschglobal/airflow/pull/2